vmdb/app/models/vm_microsoft.rb

```diff
@@ -7,4 +7,8 @@ def self.calculate_power_state(raw_power_state)
     else                        super
     end
   end
+
+  def supports_migrate?
+    false
+  end
```

Comment:
migrate is not available on redhat & SCVMM providers, available for vmware and maybe others, which is why i added a method id base class that returns true and a method in each of subclasses that do not support migrate